lib/agoneum/account/user.ex

```diff
@@ -12,17 +20,61 @@ defmodule Agoneum.Account.User do
     field :password, :string, virtual: true
     field :password_hash, :string
 
+    field :active, :boolean, default: false
+    field :guest, :boolean, default: false
+    field :admin, :boolean, default: false
```

Comment:
It shouldn't be an issue since this is a whole new app. We'd need a way to save the number of guest players for the day and let it be updated as needed, but that shouldn't be too difficult